src/applications/disability-benefits/686/config/form.js

```diff
@@ -509,10 +691,9 @@ const formConfig = {
               },
             },
             spouseAddress: _.merge(
-              address.uiSchema(
-                'Spouse address',
-                false,
-                form => form.liveWithSpouse === false,
+              createAddressUISchemaForKey(
+                'spouseAddress',
+                (...args) => !isLivingWithSpouse(...args),
```

Comment:
Could we create a `isNotLivingWithSpouse` helper, so that we can avoid specifying `(...args)` here?